lib/index.ts

```diff
@@ -108,7 +108,6 @@ async function getBrowser(
           "--disable-web-security",
         ],
         bypassCSP: true,
-        userDataDir: "./user_data",
```

Comment:
this doesn't seem to be a valid option; the types say user data dir is the first arg (we're passing `${tmpDir}/userdir` above)